sdk/core/azure-core/azure/core/tracing/_abstract_span.py

```diff
@@ -221,6 +221,7 @@ def set_http_attributes(self, request, response=None):
         :param response: The response received by the server. Is None if no response received.
         :type response: ~azure.core.pipeline.transport.HttpResponse or ~azure.core.pipeline.transport.AsyncHttpResponse
         """
+        self.kind = SpanKind.CLIENT
```

Comment:
You're changing the behavior of the mixin, why?

Reply:
Discussed offline, this is revert of:
https://github.com/Azure/azure-sdk-for-python/pull/17785/files#diff-363cd4d434eb6553b982b1473cd363d4afa4b14c36acec10f060a874f63cbe09L224

Reply:
actually - i already added the kind in the distributed tracing policy and removed it from here previously in this PR
https://github.com/Azure/azure-sdk-for-python/pull/17785
- duplicating this line here ensures backward compatibility with previosuly released versions of azure-core-tracing-opentelemetry